actions/test/run_local.ts

```diff
@@ -37,13 +37,27 @@ const main = async () => {
   const provider = await getProvider(testRuntime.context, network);
   const { chainId } = await provider.getNetwork();
 
-  provider.on("block", async (blockNumber) => {
+  // Provides a way to process an specific block p
+  const blockNumber = process.env.BLOCK_NUMBER;
+
+  const onNewBlock = async (blockNumber: number) => {
     try {
       processBlock(provider, blockNumber, chainId, testRuntime);
     } catch (error) {
       console.error("[run_local] Error in processBlock", error);
     }
-  });
+  };
+
+  if (process.env.BLOCK_NUMBER) {
```

Comment:
new env to run the actions only for a specified block (instead of running in WATCH mode)